features/cli/removing_locale.feature

```diff
@@ -0,0 +1,28 @@
+@removing_locale
+Feature: Removing locales
+    In order to deleting accidentally created locales
+    As a Developer
+    I want to have a way to remove a given locale
+
+    Background:
+        Given the store operates on a channel named "Web" with hostname "web"
+        And that channel allows to shop using "English (United States)" and "Polish (Poland)" locales
+        And it uses the "English (United States)" locale by default
+
+    @cli
+    Scenario: The developer tries to delete a not used locale
+        Given the store has a product "T-Shirt banana"
+        And this product is named "Banana T-Shirt with Minions" in the "English (United States)" locale
+        And this product has no translation in the "Polish (Poland)" locale
+        When I run a command removing the "Polish (Poland)" locale
```

Comment:
I'm not a fan of `run a command` statement, as it's strictly technical detail (and yes, I know we have it in other scenarios 😅). We have a `@cli` context to determine _how_ this removal happen.

Maybe just: `When I remove "Polish (Poland)" locale`?